lang/semgrep-grammars/src/semgrep-julia/test/corpus/semgrep.txt

```diff
@@ -219,7 +220,8 @@ end
       (vector_expression))
     (do_clause
       (parameter_list
-        (identifier))
+        (identifier
+          (semgrep_extended_metavariable)))
       (return_statement
         (identifier)))))
 
```

Comment:
maybe could have added a test with something that is not a metavariable but a valid julia identifier?